dtc-parser.y

```diff
@@ -23,6 +23,12 @@ extern void yyerror(char const *s);
 
 extern struct dt_info *parser_output;
 extern bool treesource_error;
+
+static bool is_label_relative(const char *name)
```

Comment:
Rename to `is_ref_relative` since if it returns true the given string is *not* (just) a label.  Rename `name` to `ref` would also be clearer.

Reply:
Done.